etc/profile-a-l/gconf-editor.profile

```diff
@@ -7,13 +7,13 @@ include gconf-editor.local
 # added by included profile
 #include globals.local
 
-blacklist /tmp/.X11-unix
```

Comment:
This never should have been in the profile before this PR, my mistake originally. It's a GUI app and it obviously breaks when including disable-X11.inc.

Not sure how best to proceed to fix it.

Reply:
> This never should have been in the profile before this PR, my mistake
> originally. It's a GUI app and it obviously breaks when including
> disable-X11.inc.
> 
> Not sure how best to proceed to fix it.

I'll remove it after this PR.

Reply:
Thanks!

Reply:
> I'll remove it after this PR.

Fixed it on master and rebased the PR to make the result a bit less confusing.

Anyway, good catch.